apps/pie-storybook/stories/pie-thumbnail.stories.ts

```diff
@@ -40,6 +44,34 @@ const thumbnailStoryMeta: ThumbnailStoryMeta = {
                 summary: defaultArgs.alt,
             },
         },
+        disabled: {
+            description: 'Set the disabled attribute of the thumbnail.',
+            control: 'boolean',
+            defaultValue: {
+                summary: defaultArgs.disabled,
+            },
+        },
+        hasPadding: {
+            description: 'Set the hasPadding attribute of the thumbnail.',
+            control: 'boolean',
+            defaultValue: {
+                summary: defaultArgs.hasPadding,
+            },
+        },
+        backgroundColor: {
+            description: 'Set the backgroundColor attribute of the thumbnail.',
+            control: 'text',
+            defaultValue: {
+                summary: defaultArgs.backgroundColor,
```

Comment:
we use the `defaultProps` values from the `defs` file for the summary field here to show the default values of the components. otherwise, it won't be very clear to users thinking the default values of the story vs component. 

<img width="1280" alt="Screenshot 2025-01-22 at 12 29 43" src="https://github.com/user-attachments/assets/a676e007-e896-4e13-b8ec-4896522fe683" />